services-manager/src/main/java/uk/gov/gchq/palisade/service/manager/config/ApplicationConfiguration.java

```diff
@@ -1,94 +1,94 @@
-        return runnerConfigs.entrySet().stream()
-                .map(e -> {
-                    ServiceConfiguration config = e.getValue();
-                    ProcessBuilder builder = config.getProcessBuilder();
-                    builder.directory(getServicesRoot());
-                    return new SimpleEntry<>(e.getKey(), builder);
-                })
-                .collect(Collectors.toMap(SimpleEntry::getKey, SimpleEntry::getValue));
-    }
-
-}
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.manager.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.File;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Configuration
+@EnableConfigurationProperties
+@EnableAutoConfiguration
+public class ApplicationConfiguration {
+
+    public static class ConfigurationMap {
+
+        private Map<String, ServiceConfiguration> services = new HashMap<>();
+
+        public Map<String, ServiceConfiguration> getServices() {
+            return services;
+        }
+
+        public void setServices(final Map<String, ServiceConfiguration> services) {
+            this.services = services;
+        }
+
+        @Override
+        public String toString() {
+            final StringBuilder sb = new StringBuilder("ConfigurationMap{\n");
+            sb.append('\t').append(services.entrySet().stream()
+                    .map(entry -> entry.toString().replace("\n", "\n\t"))
+                    .collect(Collectors.joining("\n\t"))).append('\n');
+            sb.append('}');
+            return sb.toString();
+        }
+    }
+
+    @Bean
+    @ConfigurationProperties(prefix = "manager")
+    ConfigurationMap configurationMap() {
+        return new ConfigurationMap();
+    }
+
+    @Bean
+    Map<String, ServiceConfiguration> serviceConfigurations(final ConfigurationMap configurationMap) {
+        return configurationMap.getServices();
```

Comment:
null

Reply:
Not an issue, it's a Spring Bean thing so should never be null